rattr/cli/toml_parser.py

```diff
@@ -1,7 +1,12 @@
 from pathlib import Path
 from typing import Any, Dict, Optional
 
-import tomli as tomllib
+try:
+    # python >= 3.11
+    import tomllib
+except ImportError:
+    # python < 3.11
+    import tomli as tomlib  # noqa: F401
```

Comment:
The F401 is legit - there's a spelling error here `tomlib` vs `tomllib`

Reply:
@habospace in case you hadn't seen this

Reply:
I did. I will update this afternoon.